pytest_html/plugin.py

```diff
@@ -271,9 +272,10 @@ def generate_checkbox(self):
                                            **checkbox_kwargs)
 
             def generate_summary_item(self):
-                self.summary_item = html.span('{0} {1}'.
-                                              format(self.total, self.label),
-                                              class_=self.class_html)
+                self.summary_item = [html.span('{0} {1}'.
+                                               format(self.total, self.label),
+                                               class_=self.class_html),
+                                     html.span('; ')]
```

Comment:
I agree it looks better with a separator, as this is what we had before the filters were introduced. Can you move this to the `for outcome in outcomes` line though, as we already have a separator there. You can also use `enumerate` and compare the index to the length of `outcomes` to avoid adding the separator for the last item.